vllm/_custom_ops.py

```diff
@@ -486,6 +487,14 @@ def cutlass_scaled_mm(a: torch.Tensor,
 
     m = a.shape[0]
     n = b.shape[1]
+
+    if current_platform.is_rocm():
+        scaled_mm_triton_module = importlib.import_module(
+            "vllm.model_executor.layers.quantization.compressed_tensors."
+            "scaled_mm_triton")
+        scaled_mm_triton = scaled_mm_triton_module.scaled_mm_triton
+        return scaled_mm_triton(a, b, scale_a, scale_b, out_dtype, bias)
```

Comment:
if `scaled_mm_triton` only handles `int8`, we should guard against use on fp8 inputs here

Reply:
It has handled any type I've used so far.  I haven't tried 64-bit types yet.